README.md

```diff
@@ -7,85 +8,72 @@ The snap packaging makes it easy to run the Chip Tool on Linux distributions.
 
-### Setup
+## Development
 
-```bash
-sudo snap install chip-tool
-```
+### Build the snap
 
-When installing from the Snap Store, the following interfaces auto-connect:
-- [`avahi-observe`](https://snapcraft.io/docs/avahi-observe-interface) to allow DNS-SD based discovery.
-- [`bluez`](https://snapcraft.io/docs/bluez-interface) for device discovery over Bluetooth Low Energy (BLE).
+Build locally for the same architecture as the host:
 
-> **Note**  
-> DNS-SD and Bluetooth depend on Avahi and Bluez.
-> To install:
-> - Ubuntu: `sudo apt install bluez avahi-daemon`
-> - Ubuntu Core: `sudo snap install avahi bluez`
+```bash
+snapcraft -v
+```
 
+Build remotely for all supported architectures:
 
-You may connect the [`process-control`](https://snapcraft.io/docs/process-control-interface) interface to allow system-wide process management.
-This is needed to grant Chip Tool access to make [sched_setattr](https://man7.org/linux/man-pages/man2/sched_setattr.2.html) system calls. This may improve the reliability of the commissioning and control operations (see [#8](https://github.com/canonical/chip-tool-snap/issues/8)).
 ```bash
-sudo snap connect chip-tool:process-control
+snapcraft remote-build
 ```
 
-### Commissioning into IP network
-Discover using DNS-SD and pair:
+### Install the built snap
+
+Install the local snap:
+
 ```bash
-chip-tool pairing onnetwork 110 20202021
+sudo snap install --dangerous *.snap
 ```
 
-where:
-
--   `110` is the node id being assigned to the app
--   `20202021` is the pin code set on the app
+### Connect interfaces
 
-### Commissioning into Thread network over BLE
-This depends on an OpenThread Border Router (OTBR) with an active network.
-For guidance on setting that up using the [OTBR snap](https://snapcraft.io/openthread-border-router), refer to [this tutorial](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap).
+When the snap is installed from the Snap Store, the `avahi-observer` and `bluez` interfaces are connected automatically.
+If you install the snap from a local file, like when you built it yourself, you need to connect these interfaces manually:
 
-Use the [OpenThread CLI](https://openthread.io/reference/cli) to obtain Thread network credential:
 ```bash
-$ sudo ot-ctl dataset active -x
-0e08...f7f8
-Done
+sudo snap connect chip-tool:avahi-observe
+sudo snap connect chip-tool:bluez
 ```
 
-Discover and pair:
+On **Ubuntu Core**, the `avahi-observe` and `bluez` interfaces are not provided by the system.
+These interfaces are provided by the [Avahi](https://snapcraft.io/avahi) and [BlueZ](https://snapcraft.io/bluez) snaps.
+To install the snaps and connect the interfaces, run:
+
 ```bash
-chip-tool pairing ble-thread 110 hex:0e08...f7f8 20202021 3840
+sudo snap install avahi bluez
+sudo snap connect chip-tool:avahi-observe avahi:avahi-observe
+sudo snap connect chip-tool:bluez bluez:service
```

Comment:
The UC related instructions can also be removed as exist on the documentation (and in in better shape).

Reply:
Ignore this. This is more about the different manual connection comments.

Reply:
A little unsure about this. Looking at the docs https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/chip-tool-commission-and-control/ it just says to install the avahi and bluez snaps, but not the commands to connect the interfaces.